scanner/src/test/kotlin/storages/ClearlyDefinedStorageTest.kt

```diff
@@ -388,7 +387,7 @@ private fun Result<List<ScanResult>>.shouldBeValid(block: (ScanResult.() -> Unit
 
         val scanResult = it.first()
         scanResult.summary.licenseFindings.find { finding ->
-            finding.location.path == TEST_PATH && "Apache-2.0" in finding.license.licenses()
+            finding.location.path == TEST_PATH && "MIT" in finding.license.licenses()
```

Comment:
Commit message should also explain that the test asset is for a different library to explain this license change.